src/models/raster/ingestion/ingestionLayer.ts

```diff
@@ -0,0 +1,34 @@
+import { LayerMetadata } from '../../layerMetadata';
```

Comment:
maybe rename the file to something more precise such as "ingestionTypes"

Reply:
done